airflow/providers/common/sql/CHANGELOG.rst

```diff
@@ -24,6 +24,11 @@
 Changelog
 ---------
 
+1.1.0
```

Comment:
Did you mean to add it without the log entry of the change?

Reply:
Yes. I need it in order to get the condition fullfilled `>= 1.1.0.*` when building prod images. I will add more description when I prepare the final release.